domenetjenester/brevbestiller/src/main/resources/dokumentmal/etterlysinntektsmelding/etterlysinntektsmelding_nn_NO.properties

```diff
@@ -1,10 +1,6 @@
 overskrift=Vi treng opplysningar fr� arbeidsgivaren din
 vi_fikk_soknaden_din_om=Vi fekk s�knaden din om {0} {1}, men manglar opplysningar for �\n\
   kunne behandle han. Du m� be arbeidsgivarane dine sende inntektsmelding til NAV.
-
 hvis_inntektsmeldingen_allerede_er_sendt=Dersom arbeidsgivar allereie har sendt inntektsmelding, kan du sj� bort fr� denne\n\
   f�respurnaden.
-
-Soknaden_kan_bli_avslatt=_S�knaden kan bli avsl�tt\n\
-  Dersom vi ikkje f�r inntektsmeldinga innan {0}, kan s�knaden din bli avsl�tt.\n\
-  Dette g�r fram av folketrygdlova � 21-3.
+info_kan_ikke_behandles=Vi kan ikkje behandle s�knaden din om {0} f�r vi har mottatt inntektsmelding fr� arbeidsgivaran(e) din(e).\nFrist for innsending er dato {1}.
```

Comment:
Tror ikke "mottatt" er helt innafor på Nynorsk? Ville skrevet "før vi har fått". Er setningen gjennomgått av noen som kan Nynorsk bra? :)

Reply:
nei, ingen ekspert som var har vært innom her! Det var godt du så for det var meningen å høre med Kristin

Reply:
Gått gjennom tekst med Kristin.